pippy/PipelineStage.py

```diff
@@ -218,7 +218,7 @@ def forward_one_chunk(
         )
 
         logger.debug(
-            f"[{self.group_rank}] Forwarded chunk {self.fwd_chunk_id}, outputs: {map_debug_info(output)}"
+            f"[{self.stage_index}] Forwarded chunk {self.fwd_chunk_id}, outputs: {map_debug_info(output)}"
```

Comment:
maybe its better to specify `Stage {stage_idx}` everywhere we log.  I could see it being still a bit confusing if you get logs saying 

"3 forwarded chunk 2 ... "

Reply:
Thanks! added.